dev/env/opensource/development.env

```diff
@@ -2,3 +2,17 @@ APP_MODE=full
 APP_ENV=development
 APP_INSTALLATION=opensource
 APP_DEV_MODE=1
+
+ZITADEL=false
+
+CLIENT_ID=269829931184226314@datalens
+CLIENT_SECRET=GRCfRsteLvSqolF122loogYU2oa7LhufWTdS6WCmiOcmSH48rqTmZOKAtLB0A4YT
+
+ZITADEL_PROJECT_ID=269829807099871242
+
+ZITADEL_URI=http://localhost:8085
+APP_HOST_URI=http://localhost:3030
```

Comment:
Let's move on to the `8080' as we discussed

Reply:
Moved